tests/tool_test.py

```diff
@@ -107,21 +107,31 @@ def patchConfigurators(mockConfigurers):
 
 class PasswdConfiguratorTest(VdsmTestCase):
     def testCheckIsConfiguredNo(self):
-        tmpfile = tempfile.mktemp()
-        with open(tmpfile, 'w') as f:
+        tmpfile = tempfile.NamedTemporaryFile(mode='w+')
```

Comment:
It would be better to collapse those 2 lines into 1: `with tempfile.Named... as f:`

Reply:
Done